store/src/java/com/zimbra/cs/mailbox/MailboxIndex.java

```diff
@@ -283,13 +282,19 @@ public boolean existsInContacts(Collection<InternetAddress> addrs) throws IOExce
                             }
                         }
                     } finally {
-                        Closeables.closeQuietly(values);
+                        try {
```

Comment:
try with resources or alternative closeQuietly in a number of places in this file.